pwndbg/commands/heap.py

```diff
@@ -80,41 +87,59 @@ def bins(addr=None):
     fastbins = main_arena['fastbinsY']
     bins = main_arena['bins']
 
+    size_t_size = gdb.lookup_type('size_t').sizeof
```

Comment:
Where do you see a pointer being used? The two fields we're skipping are `prev_size` and `size`, which are both `INTERNAL_SIZE_T` which is just `size_t`.